rdm-schema.json

```
@@ -9,11 +9,27 @@
     "manufacturer_id": {
       "$comment": "Manufacturer IDs are assigned by ESTA and are commonly expressed in hexadecimal. Users may expect to see these values in the UI as hexadecimal.",
       "title": "Manufacturer ID",
-      "description": "The manufacturer ID.",
+      "description": "The Manufacturer ID.",
       "type": "integer",
       "minimum": 0,
       "maximum": 65535
     },
+    "device_model_id": {
+      "$comment": "The Device Model ID is a 16-bit value determined by the manufacturer. Users may expect to see these values in the UI as hexadecimal.",
```

I think the second sentence should be removed, because nothing specifies that.

---

The spec says this, and that’s why I used “may”:
> The recommended method for representing the UID in text is in hexadecimal format with a colon separating the Manufacturer ID and the Device ID.
An example of such would be: mmmm:dddddddd, where mmmm is the Manufacturer ID in hexadecimal and dddddddd is the Device ID in hexadecimal.

Would you rather see “it’s possible” instead of “may”?

---

"Device Model ID" and "Software Version ID" both appear as fields under DEVICE_INFO.
The Data Description for Device Model ID says
```
This field identifies the Device Model ID of the Root Device or the Sub-Device. The Manufacturer shall not use the same ID to represent more than one unique model type.
```

---

Yes we want device_model_id. We're identifying a type of device and the software it is running, rather than a particular instance of a device. The use case is that we have a parameter definition from Device 1. If Device 2 is the same model made by the same manufacturer and running the same software, the get response would be the same thus we don't need to do a get request again, we can just used the previously retrieved parameter definition.

I'd just omit the part about displaying it as hex.

---

Lets use GitHub's features and be explicit:
```suggestion
      "$comment": "The Device Model ID is a 16-bit value determined by the manufacturer.",
```